api/server/request/views.py

```diff
@@ -61,6 +61,19 @@ def post(self):  # pylint: disable=R0201
         }
         return make_response(jsonify(response_object)), 201
 
+    @jwt_required
+    def get(self):  # pylint: disable=R0201
+        """Send GET method to requests endpoint"""
+        # get current user id
+        user_id = get_jwt_identity()
+        # Get the requests
+        get_data = all_user_requests(user_id)
+        # return response
+        response_object = {
+            "status": 'success',
+            "requests": get_data
+        }
+        return make_response(jsonify(response_object)), 200
```

Comment:
you could add a condition to check whether the `all_user_requests` returns none instead of returning an empty array you could return a more descriptive message

Reply:
Never mind 😆 , I see it already does that from the screenshots